packages/inngest/src/types.ts

```diff
@@ -1019,52 +1019,116 @@ export interface AuthenticatedIntrospection
   signing_key_hash: string | null;
 }
 
+export const functionConfigSchema = z.strictObject({
```

Comment:
Can we add a comment here as a reminder that this should be used for asserting the shape of the config, but not for feature compatibility, e.g. disallowing use of feature X with feature Y.
```suggestion
/**
 * The schema used to represent an individual function being synced with
 * Inngest.
 *
 * Note that this should only be used to validate the shape of a config object
 * and not used for feature compatibility, such as feature X being exclusive
 * with feature Y; these should be handled on the Inngest side.
 */
export const functionConfigSchema = z.strictObject({
```